base/console/src/main/java/com/netscape/admin/certsrv/Console.java

```diff
@@ -1623,6 +1631,14 @@ public static void mainImpl(String argv[]) throws Exception {
             PKILogger.setLevel(Level.INFO);
         }
 
+	String nssdb_path = null;
+        if (cmd.hasOption("d")) {
+	    nssdb_path = cmd.getOptionValue("d");
+            System.out.println("  nssdb_path specified on command line: " + nssdb_path );
+        } else {
+            System.out.println("  nssdb_path not specified on command line; using default");
+        }
+
```

Comment:
It probably can be simplified like this:

    String default_nssdb_path = FilePreferenceManager.getHomePath();
    String nssdb_path = cmd.getOptionValue("d", default_nssdb_path);
    logger.info("NSS database: " + nssdb_path);

The NSS database will appear if you specify `-v` option.